alembic/versions/20211102_1918_h.py

```diff
@@ -0,0 +1,50 @@
+"""h
+
+Revision ID: 234e66ef24b7
+Revises: 47e9177640b3
+Create Date: 2021-11-02 19:18:21.729630+00:00
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '234e66ef24b7'
+down_revision = '47e9177640b3'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
The new columns fail on a database that already has values in it because of the not null constraint. You might want to include a default value for these (e.g. an empty string or current time for date).

Reply:
I think now the changes have been done.